org.libredirect.frontends_manager.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>org.libredirect.frontends_manager</id>
+
+  <name>Libredirect Frontends Manager</name>
+  <summary>Run privacy-friendly frontends on your computer</summary>
```

Comment:
I love "privacy-friendly" but then the first thing done is punching a hole in the sandbox to allow remote executiion.